### PR TITLE
Minor cleanup in StressDivergenceTensors

### PR DESCRIPTION
**Pull request by dschwen**
Refs #8236

### CODE REVIEW

File: modules/tensor_mechanics/src/kernels/StressDivergenceTensors.C [hunk 1]
```diff
@@ -159,25 +159,29 @@ Real
...
+                                                     _grad_test[_i][_qp], _grad_phi[_j][_qp]);
 
   if (_volumetric_locking_correction)
   {
     // jacobian = Bbar^T_i * C * Bbar_j where Bbar = B + Bvol
     // jacobian = B^T_i * C * B_j + Bvol^T_i * C * Bvol_j +  Bvol^T_i * C * B_j + B^T_i * C * Bvol_j
 
     // Bvol^T_i * C * Bvol_j
-    jacobian +=  sum_C3x3 * (_avg_grad_test[_i][_component] - _grad_test[_i][_qp](_component)) * (_avg_grad_phi[_j][_component] - _grad_phi[_j][_qp](_component))/ 9.0;
+    jacobian +=   sum_C3x3 * (_avg_grad_test[_i][_component] - _grad_test[_i][_qp](_component))
```
Comment (WilkAndy) [inline modules/tensor_mechanics/src/kernels/StressDivergenceTensors.C]: extra space after "+=" ?

Are we now limiting code to 80 characters wide?

**Comment by WilkAndy** [comment]
Two general comments:

 - i had a quick look through the tests, and i don't see one that checks the Jacobian for volumetric_locking_correction = true (or use_finite_deform_jacobian either).  How do i know that the Jacobian is correct?  Maybe something for a future PR?

 - since the default for volumetric_locking_correction is "false", it suggests that users won't often want to set it true, yet it's name (including the word "correction") suggests that it'd be a good thing to use.  In this PR i think it'd be good to include a few more words in the doco to help users to make the correct choice.  Viz, expand "Set to false to turn off volumetric locking correction" slightly.  Or perhaps add something to  the tensor_mechanics section of mooseframework.org?

**Review by bwspenc — APPROVED** [review]

**Comment by bwspenc** [comment]
@WilkAndy This is something that you can really never go wrong using, and when you need it, it makes a huge difference. It is actually turned on by default in solid_mechanics. We decided to make it off by default in tensor_mechanics because many of the current users of tensor_mechanics are running problems that probably don't need it, and we didn't want to break everything.  This is something that we may want to revisit.

Good point on needing testing of the Jacobian with this option.  We need to do that. We also definitely need to document this, and it will go in the Markdown-based system.  @sveerara, maybe you can take care of those tasks when you get a chance.

**Comment by sveerara** [comment]
@WilkAndy, the jacobian tests with and without B-bar correction are in tensor_mechanics/tests/test_jacobian. These tests test the jacobian for 3D, axisymmetric and plane strain problems. For each problem type there are 6 tests -  small strain, incremental and finite strain calculators are tested with and without B-bar correction.

I will take care of the documentation part soon.

**Comment by WilkAndy** [comment]
Thanks @dschwen , @bwspenc , @sveerara for the b-bar comments.

a

**Pull request merged**